ts/components/services/NewServiceListItem.tsx

```diff
@@ -8,7 +8,6 @@ import { ProfileState } from "../../store/reducers/profile";
 import customVariables from "../../theme/variables";
 import { getEnabledChannelsForService } from "../../utils/profile";
 import ListItemComponent from "../screens/ListItemComponent";
-import { servicesRedesignEnabled } from "../../config";
 
 interface State {
   switchValue: boolean;
```

Comment:
I guess we don't need anymore to handle the activation on list item. 
Could it be removed too?